lib/cart.rb

```diff
@@ -0,0 +1,59 @@
+require_relative "menu_item"
+require_relative "menu"
+require_relative "order_item"
+require_relative "customer"
+
+class Cart
+  attr_accessor :cart
+  attr_accessor :total_cost
+
+  def initialize
+    @cart = []
+    @total_cost = 0
+  end
+
+  @@instance = Cart.new
```

Comment:
I would like to hear you thoughts why using a class variable this way is a preferable solution.